src/Component/ActionContainer.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+function ActionContainer(props) {
+  const handleAttack = () => {
+    const randomNumber = Math.floor(Math.random() * (10)) + 1
+    props.attack(randomNumber, false)
+  }
+  const handleSpecialAttack = () => {
+    const randomNumber = Math.floor(Math.random() * (20)) + 10
+    props.attack(randomNumber, true)
+  }
+  const handleHeal = () => {
+    const randomNumber = Math.floor(Math.random() * (10)) + 1
+    props.heal(randomNumber)
```

Comment:
only method should be called here, logic of the attack or heal should be in parent component

Reply:
Done